tests/test_claim.py

```diff
@@ -0,0 +1,155 @@
+from pathlib import Path
```

Comment:
in general, is it sufficient to have 1 case per method? did you check how others are handling this?

Reply:
yes, other repos are same. we just want to check the result and generate some snapshots so I think,  it will be sufficient.